api/db/repositories/customerRepository.js

```diff
@@ -34,4 +34,32 @@ exports.deleteCustomer = async (customerId) => {
     } catch (error) {
         throw new Error(error);
     }
+};
+
+exports.getAll = async (customers) => {
+    const requestedCustomer = await Customers.findAll(customers);
+
+    if(requestedCustomer === null || requestedCustomer === undefined) {
+        throw new Error('Customers table is empty. REPO');
+    }
+
+    try {
+        return Customers.findAll(customers);
```

Comment:
you do the same thing in line 40 but you assign the result to a const. You could do it a bit different 

``` js
exports.getAll = async () => {
    const allCustomers;
    try {
        allCustomers= await Customers.findAll();
    } catch (error) {
        throw new Error(error);
    }
    if (allCustomers=== null || allCustomers=== undefined) { 
        throw new Error('Customers table is empty. REPO');
    }
    
    return allCustomers;
}
```

something like that but check if it works fine.